tools/src/commands/EasDispatch.ts

```diff
@@ -187,7 +186,7 @@ async function iosBuildAndSubmitAsync() {
       [
         'pkcs12',
         '-export',
-        ...(isDarwin ? [] : ['-legacy']),
+        '-legacy',
```

Comment:
maybe we should also validate the openssl version prior to running this? so openssl > 3? (`-legacy` is not supported in 1.x)